core/domain/story_domain_test.py

```diff
@@ -208,20 +219,20 @@ def test_to_dict(self):
 class StoryDomainUnitTests(test_utils.GenericTestBase):
     """Test the story domain object."""
 
-    STORY_ID = 'story_id'
-    NODE_ID_1 = story_domain.NODE_ID_PREFIX + '1'
-    NODE_ID_2 = 'node_2'
-    SKILL_ID_1 = 'skill_id_1'
-    SKILL_ID_2 = 'skill_id_2'
-    EXP_ID = 'exp_id'
-    USER_ID = 'user'
-    USER_ID_1 = 'user1'
+    STORY_ID: Literal['story_id'] = 'story_id'
```

Comment:
Final is used where constants are not reassigned but here STORY_ID is reassigned in line 233. So, that's why Literal is used instead of Final.
( for reference: https://mypy.readthedocs.io/en/stable/final_attrs.html#final-names ) ( In the first para of Final names.)

Reply:
Let's not reassign it, lets remove it from here and only define it below, but below use lowercase characters.

Reply:
Done!